custom_components/tuya_local/devices/klarta_stor2_purifier.yaml

```diff
@@ -0,0 +1,149 @@
+name: Klarta Stor 2 air purifier
+products:
+  - id: ku6ae3y8ad6tbx3b
+    name: "KLARTA STOR 2 "
+primary_entity:
+  entity: switch
+  icon: "mdi:air-purifier"
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+secondary_entities:
+  - entity: sensor
+    class: pm25
+    dps:
+      - id: 2
+        type: integer
+        name: sensor
+        class: measurement
+        unit: ugm3
+  - entity: select
+    name: Mode
+    dps:
+      - id: 3
+        name: option
+        type: string
+        mapping:
+          - dps_val: auto
```

Comment:
When converted to fan, the fan should be given a `translation_key: fan_with_presets`, and mapped to values chosen from [available translations](https://github.com/make-all/tuya-local/blob/main/custom_components/tuya_local/translations/en.json#L169) (eg: auto => smart, manual => normal, sleep => sleep, screenoff => displayoff)